scripts/debug/validate_epub_extraction.py

```
@@ -0,0 +1,26 @@
+        reader = EpubReader(file_path)
+        chapters = reader.get_chapters()
+        print(f"Found {len(chapters)} chapters.")
+        for i, chapter in enumerate(chapters):
+            title = reader.get_chapter_title(chapter)
+            text = reader.extract_text(chapter)
+            print(f"Chapter {i+1}: {title} (Length: {len(text)} chars)")
+            # Print first 100 chars of text to verify
+            snippet = text[:100].replace('\n', ' ')
+            print(f"Snippet: {snippet}...")
+            print("-" * 20)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)